### ISSUE TITLE
Can't see my route images from other site

### ISSUE THREAD

**Issue by HarelM** [openstreetmap/openstreetmap-website#1873]
Hi,

I'm interested in presenting the user routes and allow a user to upload routes to OSM from a different UI than openstreetmap.
I'm using osm-oauth and basically most of the functionality is working, except that when I want to present a trace picture I get a 403 error message.
Here's an image link I would like to be able see after I login to my account:
https://www.openstreetmap.org/user/Harel%20M/traces/2660108/picture
My OSM alternative UI can be found here:
https://israelhiking.osm.org.il/
Login on the top right site.
Then click on the user to open a dialog with shares (non OSM related) and uploaded traces.
The second tab has the uploaded files, opening the console shows the following errors:
![image](https://user-images.githubusercontent.com/3269297/40261311-91ce598a-5b08-11e8-83f1-02b392cf935f.png)
I think it used to work in the past. I'm guessing it's probably related to security, but it would be very nice to be able to see these pictures.

**Comment by tomhughes** [comment]
Those traces are private so they're only visible to you when you're logged in - if you don't have a valid OSM session cookie, or your browser doesn't send it for a third party image load, then you won't be able to see them.

OAuth is not relevant because that's not API so not controlled by OAuth.

**Comment by HarelM** [comment]
Can you direct me to a manual or to the code that created those session cookies?

**Comment by HarelM** [comment]
As a side note @tomhughes, your behavior of "close-first-ask-questions-later" is very unfriendly and causes antagonism. **If** you want people to contribute to this project this is not the right approach. I suggest to ask if the person who opened the issue is satisfied with the answer and can close the issue himself, and only if there's no response within a given time limit then close it.

**Comment by tomhughes** [comment]
They're created when you login on the web site - they're not API that you're expected to use from a third party site, just as those images are not intended for use anywhere other than on www.openstreetmap.org - they're an implementation detail of the web site and not something intended for use elsewhere.

Yes there's nothing stopping them being used elsewhere for public traces, but if you choose to make your traces private then they'll only be visible to you when you are logged in on osm.org.

As to why I closed the ticket, well there is no bug here so what else should I do? It's not meant to be unhelpful at all - I explained what was happening and under what conditions the images would be visible and then closed the ticket.
